builds/website-distribution-pack/distribution_config.json

```diff
@@ -42,7 +42,7 @@
       "ErrorCachingMinTTL": 300
     }]
   },
-  "SmoothStreaming": true,
+  "SmoothStreaming": false,
```

Comment:
Why are we changing this?

Reply:
Unused configuration, disabling it since it won't be used in typical use cases.